packages/rancher-logging/app-readme.md

```diff
@@ -0,0 +1,2 @@
+Rancher logging allows users to configure complex logging pipelines with a few simple resources.  [ClusterFlow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/) and [ClusterOutput](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) allow
+ for collecting all logs across a cluster.  The namespaced resources [Flow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/) and [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/) allow for collecting logs from a single namespace. 
```

Comment:
I don't think so no, we never got that far. I don't think we should put too much info in there because its ending up in the header above the install stuff. For extra info (like CRD chart) we should put in either/both README or docs.